onecodex/viz/_distance.py

```diff
@@ -1,21 +1,19 @@
-import matplotlib.pyplot as plt
+import numpy as np
 import pandas as pd
-import seaborn as sns
-import scipy
-import warnings
+import altair as alt
```

Comment:
Can we do our `import`s alphabetically?

Reply:
Let's set up YAPF or Black linting on these repos... Then we can end all Python style bikeshedding once and for all ;)

Reply:
(But I agree re: alphabetical imports!)